docs/getting-started/configuration.md

```diff
@@ -0,0 +1,307 @@
+---
+id: index
```

Comment:
```suggestion
id: configuration
```

Reply:
Needs a corresponding entry in `microsite/sidebars.json` and `mkdocs.yml` for the page to show up on the sidebar. :)